WordPress/Classes/Models/Blog.m

```diff
@@ -713,6 +715,11 @@ - (BOOL)supportsTiledGallery
     return self.isHostedAtWPcom;
 }
 
+- (BOOL)supportsVideoPress
+{
+    return self.isHostedAtWPcom;
```

Comment:
Enabled only on simple WordPress.com sites for now and added [a task to the project board](https://github.com/orgs/wordpress-mobile/projects/166/views/1?layout=board) for this to eventually be changed to match the web.